test/features/standup.feature

```diff
@@ -13,43 +8,44 @@ Feature: Standup-module
             | todayMessage     | "Today I'll do this"   |
         Then the response status code should be 201
         And the response should contain:
-            | id               | 123                        |
-            | discordUser      | "eddiehubber"              |
-            | yesterdayMessage | "Yesterday I did this"     |
-            | todayMessage     | "Today I'll do this"       |
-            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | documentId | "TYPE:ID" |
 
     Scenario: search existing standup
         Given authorisation
         And make a POST request to "/standup" with:
             | discordUser      | "eddiehubber"          |
             | yesterdayMessage | "Yesterday I did this" |
             | todayMessage     | "Today I'll do this"   |
+        Then the response should contain:
+            | documentId | "TYPE:ID" |
```

Comment:
Oh we should save this ID and use it in the follow steps 🤔  - maybe for the next PR

Reply:
I thought that as well. But I decided against that because then the test would need to run in a specific order and wouldnt be independant from each other. Thats why I made it like this.

Reply:
But we can definitely add this to our list

Reply:
sure 👍